samigo/samigo-app/src/webapp/js/questionProgress.js

```diff
@@ -0,0 +1,135 @@
+<!-- Javascript for the Question Progress Panel-->
+
```

Comment:
I would suggest namespacing all this JavaScript. Not entirely necessary, but certainly a best practise.